client/ayon_core/hosts/houdini/api/lib.py

```diff
@@ -244,13 +245,14 @@ def render_rop(ropnode):
         ropnode.render(verbose=verbose,
                        # Allow Deadline to capture completion percentage
                        output_progress=verbose)
-    except hou.Error as exc:
+    except hou.Error:
         # The hou.Error is not inherited from a Python Exception class,
         # so we explicitly capture the houdini error, otherwise pyblish
         # will remain hanging.
         import traceback
         traceback.print_exc()
-        raise RuntimeError("Render failed: {0}".format(exc))
```

Comment:
I don't believe the printed exception will be added to the Publisher UI's logs - and thus, by not including the `exc` in the error or logs itself it would mean that IF the Exception itself has some information about why it failed that information will now not appear in the publish logs, which it should. So we might want to still to `log.error(f"Exception: {exc}")` or alike.

Reply:
I modified it a little bit here.
here it is https://github.com/ynput/ayon-core/pull/373/commits/e6c52f0430bd63cf8f3a469d9c7edae94d3ad8fd